scripts/superdesk-search/directives/ItemList.js

```diff
@@ -1550,6 +1550,13 @@ export function ItemList(
                 },
 
                 setSelectedItem: function(item) {
+                    if (monitoringState.selectedGroup !== scope.$id) {
+                        //If selected item is from another group, deselect all
+                        $rootScope.$broadcast('item:unselect');
+                    } else {
```

Comment:
my mistake, this should be fired only on selection items between groups... changed...